pysteps/feature/blob.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+pysteps.feature.blob
+====================
+
+Blob detection methods.
+
+.. autosummary::
+    :toctree: ../generated/
+
+    detection
+"""
+
+import numpy as np
+
+from pysteps.exceptions import MissingOptionalDependency
+
+try:
+    from skimage import feature
+
+    SKIMAGE_IMPORTED = True
+except ImportError:
+    SKIMAGE_IMPORTED = False
+
+
+def detection(
+    input_image,
+    method="log",
+    threshold=0.5,
+    min_sigma=3,
+    max_sigma=20,
+    overlap=0.5,
+    return_sigmas=False,
+    **kwargs,
+):
+    """
+    Interface to the `feature.blob_*`_ methods implemented in scikit-image. A
+    blob is defined as a scale-space maximum of a Gaussian-filtered image.
+
+    .. _`feature.blob_*`:\
+    https://scikit-image.org/docs/dev/auto_examples/features_detection/plot_blob.html
+
+    Parameters
+    ----------
+    input_image : array_like
+        Array of shape (m, n) containing the input image. Nan values are ignored.
+    method : {'log', 'dog', 'doh'}, optional
+        The method to use: 'log' = Laplacian of Gaussian, 'dog' = Difference of
+        Gaussian, ''
```

Comment:
What about 'doh'?

Reply:
good question, we should ask @pulkkins! But I assume these all come from the scikit-image documentation...